### PR TITLE
Miscellaneous typos

### PR DESCRIPTION
**Pull request by piotrdrag**
I couldn't sleep, so there we go.

### CODE REVIEW

File: toolkit/chrome/global/autocomplete.properties [hunk 1]
```diff
@@ -2,6 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-searchWithEngine=szukaj w %S
+searchWithEngine=Szukaj w %S
```
Comment (splewako) [inline toolkit/chrome/global/autocomplete.properties]: https://github.com/mozilla/gecko-dev/blame/08997000eb0686614686846a7d6bf97b2ed76fa4/toolkit//content/widgets/autocomplete.xml#L1713-L1721

I think it is used as search engine suggestion in URL bar as "SearchTerm - szukaj w SearchEngine" making this change incorrect.

**Comment by splewako** [comment]
Besides searchWithEngine string, it looks good to me,
@adrianer - r?

File: toolkit/chrome/global/autocomplete.properties (hunk 1 above)
Comment (piotrdrag) [inline toolkit/chrome/global/autocomplete.properties]: Fixed.

**Comment by adrianer** [comment]
Poza tym "GTK2 vs GTK+ 2" to r+ dla zmian w extensions/

**Comment by splewako** [comment]
https://hg.mozilla.org/l10n-central/pl/rev/23c73b201a00 and https://github.com/aviarypl/mozilla-l10n-central-pl/commit/0df952a210eb6b756f31408f7e1545f5442a807f

Thank you for the patch

**Pull request closed without merging**